### DISCUSSION TITLE
How to build for production?

### DISCUSSION THREAD

**Discussion by MrKleeblatt** [gleam-lang/gleam#3357] [Questions & support] [answered]
Hello there.

Looking through the `build` directory I noticed that my `prod/javascript` directory was empty and the `dev/javascript` directory contained all the compiled JS files. I wonder what argument I have to pass to the compiler for production and what difference it would make.

Can someone please enlighten me?
Thanks :)

**Comment by Enderchief** [comment, marked as answer]
For the JavaScript target, there is no separate "prod"/"dev" build. Currently, the prod folder is specifically for building an "Erlang Shipment" (`gleam export erlang-shipment`) but in the future the prod folder may be used for something on the JavaScript target.
